regs_every_bb/regs_every_bb.py

```diff
@@ -0,0 +1,49 @@
+'''
+Prints the registers, at every basic block within a range
+Contibutors: Toby LLoyd Davies
```

Comment:
Who is the original author?

Reply:
It was originally a gdb script. I have add the original commiter.